api/app.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+// App struct
+type App struct {
+	Router *mux.Router
+	host   string
+	port   int
+}
+
+func (a *App) getRouter() *mux.Router {
+	router := mux.NewRouter()
+	router.Handle("/servers/{type}", NewServersHandler(a)).Methods("GET")
+	router.Handle("/user/push", NewPushToUsersHandler(a)).Methods("POST")
+	return router
+}
+
+// Init starts the api
+func Init() {
+	api := &App{
+		host: "localhost",
```

Comment:
host and port should come from flags in the cmd and host should be `0.0.0.0` instead of `localhost` (example: https://github.com/topfreegames/maestro/blob/master/cmd/start.go#L76) otherwise we won't be able to call it remotely (ref: https://groups.google.com/forum/#!topic/golang-nuts/gQw-kxkoRGY)